src/main/java/org/folio/des/service/JobExecutionService.java

```diff
@@ -88,4 +120,17 @@ private JobCommand buildBaseJobCommand(Job job) {
     result.setProgress(job.getProgress());
     return result;
   }
+  private JobCommand buildResendJobCommand(Job job) {
```

Comment:
This mapping logic exactly the same with method buildBaseJobCommand()
So you can change signature of buildBaseJobCommand like this: buildBaseJobCommand(Job job, JobCommand.Type type) and use it from 2 places

Reply:
fixed